pkg/kube/kube.go

```diff
@@ -99,6 +103,23 @@ func (k *KubeClient) GetClusterStatus() ClusterStatus {
 	return Unknown
 }
 
+func (k *KubeClient) GetApplication(name string) (*wego.Application, error) {
+	cmd := []string{"get", "app", name, "-o", "json"}
+	o, err := k.runKubectlCmd(cmd)
+
+	if err != nil {
+		return nil, fmt.Errorf("could not run kubectl command: %s", err)
```

Comment:
Could we add the name of the app somewhere in the error message, it would help for debugging afterwards 🙂

Reply:
Based on this feedback, I tried an error message format like this: 

`return nil, fmt.Errorf("could not run kubectl command for %s: %s", name, err)`

To me, the error message gets a little more confusing:

`could not run kubectl command for my-app: ...`

^^ If I am the user, this makes me think that `my-app` is the problem, when really there is probably another issue.

Since the consumer has to pass in the app name as an argument already, they already have the context for the error message.